AppController/djinn.rb

```diff
@@ -5511,9 +5594,9 @@ def copy_app_to_local(appname)
     return false
   end
 
+  # This function creates the xmpp account for the app for app named 'app',
```

Comment:
"This function creates the xmpp account for the app for app named 'app'," -> 
"This function creates the xmpp account for 'app'," ?